docs/hypothesis_plugin.md

```diff
@@ -0,0 +1,46 @@
+[Hypothesis](https://hypothesis.readthedocs.io/) is the Python library for
+[property-based testing](https://increment.com/testing/in-praise-of-property-based-testing/).
+
```

Comment:
You might want to add a not saying this is supported from pydantic v1.8 and hypothesis v5.29.0 onwards only.